lib/ci-helper.ts

```diff
@@ -117,7 +117,9 @@ export class CIHelper {
         }
         await this.notes.set(messageID, mailMeta, true);
 
-        if (!this.testing && mailMeta.pullRequestURL) {
+        if (!this.testing && mailMeta.pullRequestURL &&
+            mailMeta.pullRequestURL.match(
+                /^https:\/\/github\.com\/gitgitgadget\//) ) {
```

Comment:
I'm not familiar with typescript.
Does it have `string.startsWith`? If yes, it would be 100% clearer over regex.

Reply:
> I'm not familiar with typescript.

Neither am I.

> Does it have `string.startsWith`?

What I do in such cases is scour the Javascript documentation (because Typescript is a superset of Javascript, therefore if Javascript has that method, so does Typescript). And so can you!

*clicketyclick*

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith suggests that this method is available. So I will make that change.